### PR TITLE
Update to SynapseAI v1.8.0

### PR DESCRIPTION
**Pull request by regisss**
# What does this PR do?

This PR updates the repo to SynapseAI v1.8.0. It also adds the throughput as a metric monitored for regression in the CI/CD.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: tests/baselines/albert_large_v2.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
                 "train_batch_size": 32,
                 "eval_f1": 92.2124,
                 "train_runtime": 3547.3268,
+                "train_samples_per_second": 1.0,
```
Comment (libinta) [inline tests/baselines/albert_large_v2.json:11]: is this real 1.0 for most of models? sound very low.

File: optimum/habana/transformers/trainer.py [hunk 2]
```diff
@@ -212,6 +221,74 @@ def __init__(
...
+    #                 seed=seed,
+    #             )
+    #         else:
+    #             return DistributedSampler(
+    #                 self.train_dataset,
+    #                 num_replicas=self.args.world_size,
+    #                 rank=self.args.process_index,
+    #                 seed=seed,
+    #             )
+
```
[the remark this answers is not in the document]
Comment (regisss) [reply optimum/habana/transformers/trainer.py:291]: It will be removed, it was just to launch a run with this: https://github.com/huggingface/optimum-habana/pull/152

File: tests/baselines/albert_large_v2.json (hunk 1 above)
Comment (regisss) [reply tests/baselines/albert_large_v2.json:11]: No no I'm going to update it

**Review by libinta — APPROVED** [review]
